src/neuroconv/datainterfaces/behavior/audio/requirements.txt

```diff
@@ -0,0 +1,2 @@
+soundfile==0.11.0
```

Comment:
Is there a reason to use soundfile instead of [`scipy.io.wavefile.read`](https://docs.scipy.org/doc/scipy/reference/generated/scipy.io.wavfile.read.html)?

Reply:
Well `soundfile` would support formats other than only `WAV` files, but it seems like how you install depends on the OS. I can rollback to [scipy.io.wavefile.read](https://docs.scipy.org/doc/scipy/reference/generated/scipy.io.wavfile.read.html) and revisit `soundfile` when we have examples of sound data not in `wav` files?